draft-ietf-sedate-datetime-extended.md

```diff
@@ -485,6 +481,17 @@ required to reject or perform some other error handling when
 encountering inconsistent or unrecognized suffix tags marked as
 critical.
 
+An application that encounters duplicate use of a suffix key in
+elective suffixes and does not want to perform additional processing
+on this inconsistency MUST choose the first suffix that has that key,
+i.e.,
+
+    2022-07-08T00:14:07Z[u-ca=chinese,u-ca=japanese]
```

Comment:
Ditto,
```
    2022-07-08T00:14:07Z[u-ca=chinese][u-ca=japanese]
```

Reply:
Indeed.  C&P failure on my part; thanks.

Reply:
Now #40 

In Germany, we have a term for the state of mind I must have been in:
"having tomatoes on your eyes".